tools/src/h5ls/h5ls.c

```diff
@@ -2747,9 +2757,6 @@ main(int argc, const char *argv[])
         else if (!HDstrcmp(argv[argno], "--string")) {
             string_g = TRUE;
         }
-        else if (!HDstrncmp(argv[argno], "--vfd=", (size_t)6)) {
-            preferred_driver = argv[argno] + 6;
-        }
```

Comment:
Old "drivername" parameter is retained, but looped into the new way of loading VFDs in the tools below.